### PR TITLE
Conform to pep8 rules.

### PR DESCRIPTION
**Pull request by boudewijn-tribler**
/usr/local/bin/autopep8 --in-place --max-line-length=120 --ignore=E501 --jobs=0 --recursive --pep8-passes=3 .

### CODE REVIEW

File: bloomfilter.py [hunk 1]
```diff
@@ -111,7 +113,7 @@ def _init_n_f(self, f_error_rate, n_capacity, prefix=""):
         m_size = abs((n_capacity * log(f_error_rate)) / (log(2) ** 2))
         m_size = int(ceil(m_size / 8.0) * 8)
         logger.debug("constructing bloom filter based on f_error_rate %d and %d capacity", f_error_rate, n_capacity)
-        self._init_(m_size, self._get_k_functions(m_size, n_capacity), prefix, 0L)
+        self._init_(m_size, self._get_k_functions(m_size, n_capacity), prefix, 0)
```
Comment (boudewijn-tribler) [inline bloomfilter.py:116]: Long -> Int

File: callback.py [hunk 2]
```diff
@@ -557,7 +559,7 @@ def _loop(self):
                 self._state = "STATE_RUNNING"
                 logger.debug("STATE_RUNNING")
 
-        while 1:
```
Comment (boudewijn-tribler) [inline callback.py:560]: Constant 1 -> True

File: callback.py [hunk 3]
```diff
@@ -641,7 +643,7 @@ def _loop(self):
                         with lock:
                             heappush(expired, (priority, actual_time, root_id, None, (callback[0], (result,) + callback[1], callback[2]), None))
 
-                except (SystemExit, KeyboardInterrupt, GeneratorExit), exception:
```
Comment (boudewijn-tribler) [inline callback.py:644]: except CLASS, exception -> except CLASS as exception

File: conversion.py [hunk 4]
```diff
@@ -137,7 +141,8 @@ def __init__(self, candidate, meta, offset, data, verify, allow_empty_signature)
     class EncodeFunctions(object):
         __slots__ = ["byte", "authentication", "signature", "resolution", "distribution", "payload"]
 
-        def __init__(self, byte, (authentication, signature), resolution, distribution, payload):
+        def __init__(self, byte, xxx_todo_changeme, resolution, distribution, payload):
```
Comment (boudewijn-tribler) [inline conversion.py:144]: Removed argument tuple

File: decorator.py [hunk 5]
```diff
@@ -128,8 +134,7 @@ def attach_profiler(func):
     _runtime_statistics = defaultdict(lambda: [0, 0.0])
 
     def _output_runtime_statistics():
-        entries = [(stats[0], stats[1], entry) for entry, stats in _runtime_statistics.iteritems()]
-        entries.sort()
+        entries = sorted([(stats[0], stats[1], entry) for entry, stats in _runtime_statistics.iteritems()])
```
Comment (boudewijn-tribler) [inline decorator.py:137]: code rewrite.

**Comment by whirm** [comment]
retest this please

**Comment by boudewijn-tribler** [comment]
retest this please

**Comment by whirm** [comment]
retest this please

**Pull request merged**